### PR TITLE
fixed #187

### LINKED ISSUE #187: TOX ID disappears in windows
image
Tox id sometimes disappears in windows. And cannot be copied. when tox starts it save tox id in self.id_buffer and write it to debug. but later smth happens and self.id_buffer becomes empty string. memory trouble?

### PR DESCRIPTION
**Pull request by ingvar1995**
fixed #187

### CODE REVIEW

File: src/windows/main.c [hunk 1]
```diff
@@ -1405,6 +1405,11 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PSTR cmd, int n
     //start tox thread (hwnd needs to be set first)
     thread(toxcore_thread, NULL);
 
+    //wait for tox_thread init
+    while(!tox_thread_init) {     
+        Sleep(1);
```
Comment (cebe) [inline src/windows/main.c]: better use `yieldcpu` instead of `Sleep`.

**Comment by GrayHatter** [comment]
Do you use a profile password?

**Comment by ingvar1995** [comment]
No. I updated code for encrypted profiles

**Comment by GrayHatter** [comment]
Did you test this, with the first reading I think some of this code will never run.

**Comment by ingvar1995** [comment]
after update it works even with encrypted profiles.

**Comment by GrayHatter** [comment]
merged into release-v0.6

**Pull request merged**